README.md

```diff
@@ -38,6 +38,8 @@ docker run --rm -t -v $(pwd):/home/repo coinbase/salus
 - [npm audit](docs/scanners/npm_audit.md) - Execution of [`npm audit`](https://docs.npmjs.com/getting-started/running-a-security-audit) 6.13.7 which looks for CVEs in node module dependencies.
 - [yarn audit](docs/scanners/yarn_audit.md) - Execution of [`yarn audit`](https://yarnpkg.com/lang/en/docs/cli/audit/) 1.22.0 which looks for CVEs in node module dependencies.
 - [PatternSearch](docs/scanners/pattern_search.md) - Execution of [`sift`](https://sift-tool.org/docs) 0.9.0, looks for certain strings in a project that might be dangerous or could require that certain strings be present.
+- [Cargo Audit](docs/scanners/cargo_audit.md) - Execution of [Cargo Audit](https://github.com/RustSec/cargo-audit) Audit Cargo.lock files for crates with security vulnerabilities reported to the RustSec Advisory Database
```

Comment:
could you add the cargo audit version, like the other scanners?

Reply:
Sounds good, I've locked the version of cargo audit and added that version info (0.12.0) to the Readme